Examples/HttpAnalyzer/main.cpp

```diff
@@ -398,9 +399,18 @@ void analyzeHttpFromPcapFile(std::string pcapFileName)
 		EXIT_WITH_ERROR("Could not open input pcap file");
 
 	// set a port 80 filter on the reader device to process only HTTP packets
-	PortFilter httpPortFilter(80, SRC_OR_DST);
-	if (!reader->setFilter(httpPortFilter))
-		EXIT_WITH_ERROR("Could not set up filter on file");
+	if (GlobalConfig::getInstance().setDstFilterPort)

```

Comment:
instead of add a `GlobalConfig` object, why can't we simply add the port as additional parameter to `analyzeHttpFromPcapFile()`?

This parameter can have a default value of `80`